src/AColorResources.h

```diff
@@ -1,3 +1,15 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  AColorResources.h
```

Comment:
It's included from AColor.cpp.
It contains RGB value of new spectrogram colormap.

Reply:
Now I see, you added the #include in the first commit.  I thought it would be in the same commit that added AColorResources.h.

This means the first commit can't compile. I prefer that each commit in a commit history should compile.

After more complete review, I would like the new commits to be squashed, but it is also good to leave the first refactoring commit separated, and fixed so it compiles.